modules/test-framework-manager/lambda/test_framework_manager.js

```diff
@@ -18,19 +18,18 @@ let env_name;
 let env_color;
 let deploymentType; // values ECS (CodeDeploy), SAM, AppMesh
 let taskToken;
+let app_config = {};
 
 exports.handler = function (event, context, callback) {
   console.log('event', event);
   if (event.DeploymentType == "AppMesh") {
-      deploymentType = "AppMesh";
-      taskToken = event.TaskToken;
+    deploymentType = "AppMesh";
+    taskToken = event.TaskToken;
   }
-  //if (event.hasOwnProperty("DeploymentId")) {
   else if (event.DeploymentId) {
```

Comment:
you wanted to keep the prop check with hasOwnProperty() in today discussion, but you removed it. Any specific reason?

Reply:
I took a chance to check if it is still needed and it worked without , so it is OK to remove